db/seeds/3-orders-and-intents.js

```diff
@@ -0,0 +1,51 @@
+exports.seed = function (knex, Promise) {
+  const order = {
+    consumerAgentId: 2,
+    supplierAgentId: 3,
+    adminAgentId: 8
+  }
+
+  return knex('orders').insert(order).returning('id')
+    .then(([orderId]) => {
+      console.log(orderId)
+      const orderIntents = [{
+        agentId: 8,
+        desiredQuantity: 11,
```

Comment:
i guess i missed this in a previous review, but all numbers should be strings, so `desiredQuantity` should be a string. not specific to this pull request, but figured worth mentioning.